functions/source/rest_client.py

```diff
@@ -294,6 +299,35 @@ def create_workspaces(account_id, workspace_name, deployment_name, aws_region, c
        
             response.update(response_policy)
             print(response)
+
+            # DEFAULT CLUSTER
+            if default_cluster:
+                
```

Comment:
Seems like this is only run if `hipaa_parm` is 'Yes'? Should put this logic outside of the hipaa_parm if statement.

Reply:
Do you konw what is `hippa_param` is?

Reply:
I think it's a parameter that's passed in on the standard quickstart template (https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/create/parameters). Don't think it would be passed in for the trial template.
<img width="1016" alt="Screen Shot 2021-06-14 at 11 18 33 AM" src="https://user-images.githubusercontent.com/13651069/121939814-479c8200-cd02-11eb-9196-1fe12b61d44d.png">